### PR TITLE
added functionality pressed button

### PR DESCRIPTION
**Pull request by PalmiraBenavente**
- MVP implementation
- Added contract, model, presenter and view files and folders
- Added String for the Toast message in values -> strings
- Added viewBinding in Gladle.

image
image

- Added base folder
image

### CODE REVIEW

**Review by mauroLonghi — APPROVED** [review]

File: app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+public class ParkingView extends ActivityView implements ParkingContract.MainActivityView {
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    @SuppressLint("StringFormatInvalid")
```
Comment (lucianoeli) [inline app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java]: is this really necesary? delete if not please

File: app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.example.parkingsystem.mvp.view;
+
+import android.app.Activity;
+import android.app.FragmentManager;
+import android.content.Context;
+
```
Comment (lucianoeli) [inline app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java]: delete this emptty lines please.
you can configure AS fort that

Comment (lucianoeli) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java]: check other files too please

File: app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package com.example.parkingsystem.mvp.view;
```
Comment (SartiMau) [inline app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java]: this class should be in a base package inside the view package

File: app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java (hunk 1 above)
Comment (SartiMau) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java]: Please delete this SuppressLint try to fix the warning in another way. I'm not sure if it is necessary

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java (hunk 1 above)
Comment (PalmiraBenavente) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java]: Done

File: app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java (hunk 2 above)
Comment (PalmiraBenavente) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java]: Done

File: app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java (hunk 3 above)
Comment (PalmiraBenavente) [reply app/src/main/java/com/example/parkingsystem/mvp/view/ActivityView.java]: Done

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Pull request merged**